backend/fms_core/services/samplesheet.py

```diff
@@ -78,7 +78,10 @@ def get_samplesheet(container_kind, placement):
     return out_stream.getvalue(), errors, warnings
 
 def _get_Sample_ID(sample_alias, derived_sample_id):
-    return f"{sample_alias}_{derived_sample_id}"
+    Sample_ID = f"{sample_alias}_{derived_sample_id}"
+    if len(Sample_ID) > 100:
+        raise Exception(f"Sample_ID '{Sample_ID}' is too long. It must be less than or equal to 100 characters.")
```

Comment:
Not sure it is the best way to deal with that ... i would rather cut-off the sample_alias part short of the extra length... it would be unique but the sample_name would be messy but if someone use 150+ characters for their sample name they deserve that ;)

Reply:
I mention that because if you let it as an exception the lab will have to create the samplesheet by themselves and cut the sample_ids manually ... this is worse than having sample longer than 100. I expect the solution to the issue is to truncate the name while keeping the derived_sample_id part for uniqueness.